src/main/java/org/apache/commons/net/ftp/FTP.java

```diff
@@ -215,6 +215,11 @@
 
     private static final String modes = "AEILNTCFRPSBC";
 
+    /**
+     * An empty immutable {@code String} array.
+     */
+    private static final String[] EMPTY_STRING_ARRAY = new String[0];
```

Comment:
-1 We don't need duplicate constants in the same package all over the place. Just make one of them _package private_ like the one in `FTP` and reuse it.

Reply:
Ok. Changed. What do you think about this applied approach?